appveyor.yml

```diff
@@ -0,0 +1,34 @@
+version: 0.1.{build}
+
+environment:
+  matrix:
+    - nodejs_version: 8
+
+platform:
+  - x64
+
+#services:
+
+cache:
+  - '%USERPROFILE%\.electron'
+  - C:\Users\appveyor\AppData\Local\electron-builder\cache
+  - C:\projects\image-exporter-vm18n\node_modules
```

Comment:
@JamesCropcho I don't know why these paths are listed in the cache. Have you found any documentation about this?

I've noticed `electron-builder` uses [these paths](https://github.com/electron-userland/electron-builder/blob/8f19ea99a9cc6fa4796f8057c268541a7b7f63b4/appveyor.yml#L4-L7)

Reply:
@n-riesco  I am unsure why I chose those paths, however I feel most confident copying from Electron-Builder, so I have replaced my cache entries with theirs.

AppVeyor has not been running builds for the past fifteen minutes, so I am unable to verify a lack of errors at present.